include/Page/Base.php

```diff
@@ -298,7 +299,7 @@ public function add_metabox( $metabox ) {
 			return;
 		}
 
-		$this->metaboxes[] = $metabox;
+		$this->metaboxes[ $metabox->get_metabox_slug() ] = $metabox;
```

Comment:
Any specific reason for adding the key to the array?

Reply:
> Any specific reason for adding the key to the array?

I actually tried to hook the AJAX requests in the Metabox class initially. At that point, the AJAX requests didn't work and move them to a separate file. But then, I forgot to remove these methods. Now I've removed them.